File: ESIEA/Livrables-ESIEA_TD COO_gestionResto/done/37. TDN ESIEA 2A BENEDETTI/TDN ESIEA 2A BENEDETTI - Restaurant/src/Products/platdujour.cpp
#include "platdujour.h"

PlatDuJour::PlatDuJour(const unsigned int &price, const double &tva) :
    Produit(price, tva)
{
}

PlatDuJour::~PlatDuJour()
{
}

std::string PlatDuJour::getName() const
{
    return "Plat du jour";
}

File: ESIEA/Livrables-ESIEA_TD COO_gestionResto/done/16. TDN ESIEA 2A REMY - RANAIVO/TDN ESIEA 2A REMY - RANAIVO/src/resto/PlatDuJour.java
package resto;

public class PlatDuJour extends Plats{
	int blanquette;
	
	public PlatDuJour(){
		blanquette = 12;
	}
	void affichagePDJ(){
		System.out.println("******PLAT DU JOUR******");
		System.out.println("1 - Blanquette : "+blanquette+"€");
	}
}